packages/sign_in_with_apple/example/macos/Podfile.lock

```diff
@@ -0,0 +1,16 @@
+PODS:
+  - FlutterMacOS (1.0.0)
+
+DEPENDENCIES:
+  - FlutterMacOS (from `Flutter/ephemeral/.symlinks/flutter/darwin-x64`)
+
+EXTERNAL SOURCES:
```

Comment:
Ah, while the `sign_in_with_apple` plugin once was here, it now disappeared again. That would explain the above error.

Reply:
Ah, for macOS we need the new format in pubspec.yaml, while that didn't work for iOS for me…